core/lib/object_store/src/retries.rs

```diff
@@ -28,6 +28,18 @@ impl Request<'_> {
         self,
         store: &impl fmt::Debug,
         max_retries: u16,
+        f: F,
+    ) -> Result<T, ObjectStoreError>
+    where
+        Fut: Future<Output = Result<T, ObjectStoreError>>,
+        F: FnMut() -> Fut,
+    {
+        self.retry_internal(max_retries, f).await
+    }
+
+    async fn retry_internal<T, Fut, F>(
+        &self,
+        max_retries: u16,
```

Comment:
JFYI: this is an artifact that I'm gonna revert.

Reply:
Reverted [here](https://github.com/matter-labs/zksync-era/pull/2980/commits/a559c401a66b170c4ae2c15e205306a448c28e21). ✅